### ISSUE TITLE
Add four season value?

### ISSUE THREAD

**Issue by Nastras** [naofireblade/homebridge-weather-station-extended#10]
Hello,

is it possible that you add a value for the four seasons in your plugin?

Would be great for certain functions.

Greeting Nastra

**Comment by naofireblade** [comment]
This would be easy to implement but I think it does not belong into this plugin. I could create a new plugin or maybe add it to my homebridge-logic plugin.

**Comment by Nastra888** [comment]
I think it would fit into the plugin, but if you would rather make your own for it I would find it great of you.

**Comment by Nastras** [comment]
Hello, I wanted to ask if you can say when you have time to implement it?

**Comment by naofireblade** [comment]
Hi Nastras, I will try to implement it on the weekend.

**Comment by Nastras** [comment]
Wonderful, thank you!

Do you already know in which plugin you implement it?

**Comment by naofireblade** [comment]
Probably in a new tiny plugin. I still think it does not fit in the weather plugin and the logic plugin has a lot of overhead (UI for adding / removing variables) that not everybody wants in his homekit setup.

**Comment by naofireblade** [comment]
Here you go :) https://github.com/naofireblade/homebridge-seasons

Tell me if you have any ideas for further improvement.

**Comment by Nastras** [comment]
Hello naofireblade,

Thank you for the implementation of the idea. I have already installed the plugin and it does exactly what I imagined.

If I still have ideas for the plugin, I'll let you know.

Many Thanks !


Da ich gesehen habe das du als Standort Deutschland angegeben hast probiere ich es einfach mal auf Deutsch. Echt super das du es so schnell umgesetzt hast, irgendwie war diese Funktion noch eine Lücke in der Homebridge Landschaft.
Wenn mir noch Sachen zur Weiterentwicklung einfallen für das Plugin melde ich mich natürlich wieder.

Eventuell hast du ja Lust mal im SmartApfel HomeKit Forum vorbei zu schauen auf eine Runde SmalTalk, dort gibt es auch schon einen Thread zu deinem neuen Plugin .-)

http://forum.smartapfel.de/forum/thread/1067-plugin-%E2%80%9Eseason%E2%80%9C/?postID=20174#post20174

Gruß Nastra
